pkgs/development/libraries/libfive/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, eigen3_3, zlib
+, libpng, boost, qt5, guile
+}:
+
+stdenv.mkDerivation rec {
+  name = "libfive-${version}";
+  version = "755f7fed";
```

Comment:
If the package doesn't have frequent releases, usually the date of the commit is used as the version then, in YYYY-MM-DD format.

Reply:
Good to know. Doing that and the below.